watertap/unit_models/nanofiltration_DSPMDE_0D.py

```diff
@@ -1386,23 +1515,23 @@ def initialize_build(
                     raise InitializationError(
                         "The property package failed to solve during initialization."
                     )
-        check_solve(
-            res,
-            checkpoint="Solve in Initialization Step 3",
-            logger=init_log,
-            fail_flag=fail_on_warning,
-        )
+        # check_solve(
+        #     res,
+        #     checkpoint="Solve in Initialization Step 3",
+        #     logger=init_log,
+        #     fail_flag=fail_on_warning,
+        # )
         # Release Inlet state
         self.feed_side.release_state(flags_feed_side, outlvl)
-        # Rescale any badly scaled vars
-        if automate_rescale:
-            badly_scaled_vars = list(iscale.badly_scaled_var_generator(self))
-            if len(badly_scaled_vars) > 0:
-                init_log.warn(
-                    f"After solve: {len(badly_scaled_vars)} poorly scaled "
-                    f"variable(s) will be rescaled so that each scaled variable value = 1"
-                )
-            self._automate_rescale_variables()
+        # # Rescale any badly scaled vars
+        # if automate_rescale:
+        #     badly_scaled_vars = list(iscale.badly_scaled_var_generator(self))
+        #     if len(badly_scaled_vars) > 0:
+        #         init_log.warn(
+        #             f"After solve: {len(badly_scaled_vars)} poorly scaled "
+        #             f"variable(s) will be rescaled so that each scaled variable value = 1"
+        #         )
+        #     self._automate_rescale_variables()
```

Comment:
You re-scaled prior to initialization, but not after. Should this be active?

Reply:
I might take the automate_rescale out all together since it is barely being used right now and was able to stabilize the model without it (for the subset of cases I tried).